internal/provider/resource_gitlab_managed_license.go

```diff
@@ -47,8 +47,8 @@ var _ = registerResource("gitlab_managed_license", func() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     false,
-				ValidateFunc: validation.StringInSlice([]string{"approved", "blacklisted"}, true),
-				Description:  "Whether the license is approved or not. Only 'approved' or 'blacklisted' allowed.",
+				ValidateFunc: validation.StringInSlice([]string{"approved", "blacklisted", "allowed", "denied"}, true),
+				Description:  "Whether the license is approved or not. 'allowed' or 'denied' are acceptable values. 'approved' and 'blacklist' are accepted, but removed in GitLab 15.0",
```

Comment:
Please use this function:

https://github.com/gitlabhq/terraform-provider-gitlab/blob/49db985b79f667da9244a57d9ff4f107f337b506/internal/provider/util.go#L17

To keep the docs and validation in sync. I also suggest that you use the same wording for docs as other users of that function.

Reply:
Yep, I'll get that done at the same time as I fix up what armsnyder is asking for.

Reply:
Sorry, should have left a comment, but this was fixed in the last commit.